source/MRMesh/MRObjectLoad.h

```diff
@@ -23,8 +23,11 @@ MRMESH_API tl::expected<ObjectPoints, std::string> makeObjectPointsFromFile( con
 /// loads distance map from given file in new object
 MRMESH_API tl::expected<ObjectDistanceMap, std::string> makeObjectDistanceMapFromFile( const std::filesystem::path& file, ProgressCallback callback = {} );
 
+// check if exist any supported files in folder and subfolders
+MRMESH_API bool isAnySupportedFilesInSubfolders( const std::filesystem::path& folder );
```

Comment:
Correct in English "Is there any support file" or "Are there any supported files"

Reply:
fixed

Reply:
And function names as well, please. At least `is`-`File`